tests/std/bool_test.go

```diff
@@ -34,7 +34,7 @@ func TestStdBool(t *testing.T) {
 	require.NoError(t, err)
 	for name, protocol := range dsns {
 		t.Run(fmt.Sprintf("%s Protocol", name), func(t *testing.T) {
-			if conn, err := GetStdDSNConnection(protocol, useSSL, "false"); assert.NoError(t, err) {
+			if conn, err := GetStdDSNConnection(protocol, useSSL, nil); assert.NoError(t, err) {
```

Comment:
Is this a relevant change? If not, let's avoid mixing several changes in a single PR.

Reply:
As mentioned in a internal message, I couldn't avoid it.